install/operator/cmd/manager/main.go

```diff
@@ -45,7 +45,6 @@ func printVersion() {
 
 func main() {
 	configuration.TemplateConfig = pflag.StringP("template-config", "t", "/conf/template-config.yml", "Path to template config used for installation")
-	configuration.AddonsDirLocation = pflag.StringP("addons", "a", "", "Path to the addons directory used for installation")
```

Comment:
Don't we want this to be configurable? I'm even thinking that this should be a `StringSliceP` so one could specify multiple addon directories. Use cases like the ones we ship vs the one you'd like to test and mount via volume?

Reply:
well no, the problem before is that we could not enable/disable individual components, so we either we include the `addons` dir (and all its content) or we didn't. Now that we can toggle components we can always load the `addons` content

Reply:
I would agree with @zregvart in this instance. In my experience there are always reasons for customers to want to install addons of their own that cannot be included openly (secure/sensitive in nature being one example). So if the addons directory can be configured it really can only enhance in my view.

Reply:
I would not feel comfortable making our operator an open planform for install any user created content.  So I'd rather not allow user created resources to be managed by our operator.  Remember, the smaller the surface area of what can be modified by the user, the better it is for us in terms of support and future flexibility to change this as we want.  Once a user start using something and we want to change it later, now we need to think about how can we migrate him, or have a compatibility story.